src/Task/PhpCsFixerV2.php

```diff
@@ -72,7 +72,15 @@ public function run(ContextInterface $context)
         $arguments->addOptionalArgument('--allow-risky=%s', $config['allow_risky'] ? 'yes' : 'no');
         $arguments->addOptionalArgument('--cache-file=%s', $config['cache_file']);
         $arguments->addOptionalArgument('--config=%s', $config['config']);
-        $arguments->addOptionalCommaSeparatedArgument('--rules=%s', $config['rules']);
+
+        $rules = $config['rules'];
+        // JSON-encode rules if specified as a map.
+        if (array_values($rules) !== $rules) {
+            $arguments->addOptionalArgument('--rules=%s', json_encode($rules));
+        } else {
```

Comment:
Object callisthenics, or whateverthefuck, complains about this. I am not used to being forbidden from writing `else` statements. How "else" should I write this?